### ISSUE TITLE
io.display.table is missing image option

### ISSUE THREAD

**Issue by homoky** [interval/interval-node#11]
`/workspaces/clavard/node_modules/@interval/sdk/dist/types.d.ts`:

```
export declare type TableColumnResult = {
    label: string | number | boolean | null | Date | undefined;
    value?: TableCellValue;
    url?: string;
    action?: string;
    params?: z.infer<typeof serializableRecord>;
} | TableCellValue;
```

`"version": "0.28.0"`

**Comment by jacobmischka** [comment]
![image](https://user-images.githubusercontent.com/3939997/190903749-a999cdea-38a3-4437-94ab-0c5575581ba1.png)

![image](https://user-images.githubusercontent.com/3939997/190903730-e76c7df9-7376-4fe8-99ef-61587e93315b.png)

Coming this week! 🙊

**Comment by alexarena** [comment]
Hey @homoky, we shipped this in [SDK version 0.29](https://github.com/interval/interval-node/releases/tag/v0.29.0). Thanks for submitting the issue 😃
